physics/module_bl_mynn.F90

```diff
@@ -3003,10 +2865,16 @@ SUBROUTINE  mym_turbulence (                                &
        !IF ( sm(k) > sm25max ) sm(k) = sm25max
        !IF ( sm(k) < sm25min ) sm(k) = sm25min
        !sm(k) = Prnum*sh(k)
-       slht  = zi*0.1
-       wtpr  = min( max( (slht - zw(k))/slht, 0.0), 1.0) ! 1 at z=0, 0 above sfc layer
-       Prlim = 1.0*wtpr + (1.0 - wtpr)*Prlimit
-       sm(k) = MIN(sm(k), Prlimit*Sh(k))
+
+       !surface layer PR
+       !slht  = zi*0.1
```

Comment:
Consider whether commenting out or deleting is more appropriate here. Git history can always save stuff like this if you want to make the code "prettier".

Reply:
Yeah, some things I need to keep until I think about it more. I really only want to delete things I know I want out permanently.